src/domain/timeline/calendar/views/EventForm.tsx

```diff
@@ -122,3 +123,11 @@ const EventForm = ({ typeOptions, isSubmitting, actions }) => {
 };
 
 export default EventForm;
+
+type EventFormProps = {
```

Comment:
Can we put this type before the component (like in every other place) or in a separate file (as you prefer)?

Reply:
Agree! Moving it into a separate file.